packages/peregrine/lib/talons/Checkout/useAddressForm.js

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+
+/**
+ * Returns values used to render an AddressForm component.
+ * @param {Object} props
+ * @param {Object} props.fields an array of fields to reduce over for initial values
+ * @param {Object} props... TODO complete these
+ */
+export const useAddressForm = props => {
+    const {
+        onCancel,
+        countries,
```

Comment:
@jimbo `countries` would normally come from context so I wanted to make sure to export it from the talon even though nothing happens to the incoming prop here. Thoughts?